src/instructlab/eval/exceptions.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+class EvalError(Exception):
+    """
+    Parent class for all of instructlab-eval exceptions
+    """
+
+
+class ModelNotFoundError(EvalError):
+    """
+    Exception raised when model is not able to be found
+
+    Attributes
+        model   model that is being operated on
+    """
+
+    def __init__(self, model) -> None:
+        super().__init__()
+        self.model = model
```

Comment:
model_path

Reply:
This one I think would remain `model` since the exception refers to a model and not to a path?

Reply:
Is it the model name?  Whatever it was, it couldn't be found.  I was guessing the only thing that would be available to the calling code is that path.

Reply:
I tweaked the implementation to take the path and dynamically determine the model name for the exception message